### DISCUSSION TITLE
Allow HTML for tooltip content - Icon Button

### DISCUSSION THREAD

**Discussion by GamendeBrian** [filamentphp/filament#14065] [Ideas]
I am using the Icon Button blade component like so:

```html
<x-filament::icon-button
    icon="heroicon-o-information-circle"
    size="xs"
    color="danger"
    tooltip="
        {{ $deadline->due_date->format('d/m') }}<br />
        {{ $deadline?->project?->title }}<br />
        {{ $deadline?->title }}
    "
/>
```

This currently doesn't seem to work because the tooltip options doesn't have the allow HTML option enabled.

This can be enabled like so:
`x-tooltip.html="true"`

Can this be added by default? Or add a prop like allowHtml?

If yes, can this also be done for other tooltip component etc?

Thanks!

**Comment by KONEY** [comment]
I'm using ->hintIcon('heroicon-m-information-circle', tooltip: 'html text'); and this also applies to hint.blade.php. Would be nice to support HTML in every tooltip by default.

Thanks

**Comment by borex** [comment]
It would be great to have the tooltip render as HTML, but till then you can copy the hint.blade.php to resources\views\vendor\filament-forms\components\field-wrapper folder and modify it by adding allowHTML: true to the tooltip. This worked for me.

**Reply by GamendeBrian** [reply]
Yeah, I did this as well for now. It would be nice if they baked this in.
